Seccion 24 lista

KevinMed25 opened the pull request.

KevinMed25 committed:
avance seccion 24

KevinMed25 committed:
Seccion 24 lista

jfer1990 reviewed (changes requested):
Kevin, tienes un error en el archivo foeach.php , debe deberse a un problema con la función foreach, revisa cuál pueda ser el error y pushea tu corrección, en el commit pon cuál fue el problema porfa.
Tienes un problema igual con el archivo 15 de funciones, revisa la sumatoria que realizas en sumar2 y corrigelo igual porfavor
Tienes otro problema relacionado a las variables en el archivo 18 de json

KevinMed25 committed:
Correción sección 24

El motivo del problema en el archivo 14-foreach.php, estaba en la función foreach, el error estuvo porque al final hice un comentario y no me fije que en ese pedazo de código correspondia a html, y la forma de comentar es distinta, por eso me mandaba el error. También me confundi con el nombre del parametro, puse producto en plural.

En el 15 de funciones, el array lo puse solo para corroborar que permite la funcion permite parametros de distinto tipo, pero ya lo modifique y ahora si realiza la suma. 

En el archivo del json, igual me confundi con el nombre de la variable, coloqué productos en vez de producto.
